### PR TITLE
Add OVN logLevel parameters to `kind.sh`

### PR DESCRIPTION
**Pull request by roytman**
Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

Add OVN/OVS logLevel parameters to the `kind.sh` script

### CODE REVIEW

File: contrib/kind.sh [hunk 1]
```diff
@@ -143,10 +194,18 @@ print_params()
...
+     echo "OVN_LOG_LEVEL_NORTHD = $OVN_LOG_LEVEL_NORTHD"
+     echo "OVN_LOG_LEVEL_NB = $OVN_LOG_LEVEL_NB"
+     echo "OVN_LOG_LEVEL_SB = $OVN_LOG_LEVEL_SB"
+     echo "OVN_LOG_LEVEL_CONTROLLER = $OVN_LOG_LEVEL_CONTROLLER"
+     echo "OVN_LOG_LEVEL_NBCTLD = $OVN_LOG_LEVEL_NBCTLD"
      echo ""
 }
 
-parse_args $*
+parse_args "$@"
```
Comment (aojea) [inline contrib/kind.sh:208]: this is needed to deal with this args `-vconsole:info -vfile:info"` right?

can it break something?

Comment (roytman) [reply contrib/kind.sh:208]: Yeah, it's needed to transfer the strings with spaces. 

> can it break something? 
I deployed kind a couple of times, it works correctly unless you put your arguments in quotes.

Comment (danielmellado) [reply contrib/kind.sh:208]: I guess it's ok for me, maybe worth a small comment there but not really a blocker for me

**Review by danielmellado — COMMENTED** [review]
/lgtm

**Comment by danielmellado** [comment]
@trozet could you please PTAL as well?

File: contrib/kind.sh (hunk 1 above)
Comment (roytman) [reply contrib/kind.sh:210]: A comment was added.

**Pull request merged**